connect/runtime/src/main/java/org/apache/kafka/connect/runtime/rest/util/SSLUtils.java

```diff
@@ -47,7 +47,7 @@ public static SslContextFactory createSslContextFactory(WorkerConfig config) {
     public static SslContextFactory createSslContextFactory(WorkerConfig config, boolean client) {
         Map<String, Object> sslConfigValues = config.valuesWithPrefixAllOrNothing("listeners.https.");
 
-        SslContextFactory ssl = new SslContextFactory();
+        SslContextFactory.Server ssl = new SslContextFactory.Server();
```

Comment:
I think you have to check the `client` boolean to decide which factory to use, right?